README.md

```diff
@@ -1,3 +1,16 @@
+---
```

Comment:
Please remove this ugly table. It is related to dev portal search results, but instead of having it shoved in here, talk with @nokafor about getting a way to index these which works better for the projects.

Reply:
Yeah we need to remove that from they typescript version as well

Reply:
removed table